ConsoleApp1/Animal.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    public class Animal
+    {
+        public string name;
+        public int age; 
+        public float size;
+
+        public Animal() { name = "Unknown"; age = 0; size = 0; }
+        public Animal(string name, int age, float size)
+        {
+            this.name = name;
+            this.age = age;
+            this.size = size;
+        }
+
+        static public void Show(string name, int age, float size) =>  Console.WriteLine($"Наименование: {name}, возраст: {age} лет, размер: {size}.");
+
+        static public void GetGrow(ref float animalSize, int foodSize)
+        {
+            animalSize = (float)(animalSize+(foodSize*0.3));
+            Console.WriteLine($"Животное увеличилось на {(float)(foodSize * 0.3)}");
```

Comment:
nit: мини оптимизация - давайте считать значения один раз
(если другой разрабочик будем менять коэффициент, как тут, то чтобы менял в одном месте)

```suggestion
            var delta = (float)(foodSize * 0.3);
            animalSize = (float)(animalSize+delta );
            Console.WriteLine($"Животное увеличилось на {delta }");
```

и по поводу приведения типов, не знаю специально так написали или нет, но реально получается
```
foodSize - int
0.3 - это Double

foodSize  * 0.3 - это double
и вы приводите его к float 
```

чтобы не было преобразований, нужно вместо литерала `0.3` написать `0.3f` - тогда вычисления сразу будут идти во float